app/src/ui/dispatcher/dispatcher.ts

```diff
@@ -2704,11 +2705,56 @@ export class Dispatcher {
     }
   }
 
-  /**
+  /*
```

Comment:
I think this change should be reverted, AFAIK the initial `/**` is important to parse documentation from comments!
```suggestion
  /**
```

Reply:
Yep. It is. I had some merge conflicts here when updating my branch. Didn't release it did this. Thanks